✨(frontend) hide teacher dashboard link (#2114)
openfun/richie: CHANGELOG.md

## [Unrealeased]

### Added

>  

Reviewer: Remove this extra space

### Changed

> - Teacher dashboard access is restricted to user that are linked to a course 
>   or an organization.

Reviewer: ```suggestion
- Teacher dashboard access is restricted to user which has access to a course 
  or an organization.
```
